x/pairing/keeper/stakeModify.go

```diff
@@ -29,7 +30,8 @@ func (k Keeper) BurnClientStake(ctx sdk.Context, chainID string, clientAddressTo
 
 		if clientEntry.Stake.IsLT(k.MinStakeClient(ctx)) {
 			// if user doesn't have enough stake to stay staked, we will unstake him now
-			logger.Info("unstaking client", clientEntry.Address, "insufficient funds to stay staked", clientEntry.Stake)
+			details := map[string]string{"stake_entry": clientEntry.String()}
+			utils.LogLavaEvent(ctx, logger, types.ConsumerInsufficientFundsToStayStakedEventName, details, " insufficient funds to stay staked.")
```

Comment:
there is an extra space at the beginning of each description

Reply:
I saw that when it actually prints the message, it prints it with no space after the end of the previous sentence, so these extra spaces are on purpose.
Remove them?

Reply:
after talking with Yarom, I'll keep it